АпдейтБоёвки

Jordanstolethepreciousslam opened the pull request:
Серия твиков касающаяся обделенным вниманием механикам культа, генокрада, боевых исскуств и тени

Commit:
Твики баланса
-Станпрод
--Урон выносливости уменьшен до 20(5 ударов до стамина-крита)
--Убрана пререзарядка станпрода из-за которой он оглушал в 3 раза реже задуманного
-Культ
--Оглушение
---Оглушение снижено с 5 тиков до 2
---Наносит 30 урона выносливости сразу
---Наносит еще 100 урона выносливости в течение 10 секунд после применения
-Предатель
--Mind Batterer
---Минимальный стамина урон повышен с 20 до 25

Commit:
Ребаланс БИ, генокрада и заклинаний культа

 CQC
-Цена повышена с 12 до 14
-Больше не может быть получен по скидке
-Шанс выхватывания оружия при дизарме снижен до 50%
    Комбинации
     Leg Sweep
         Длительность оглушения с ~5 до ~0.5 секунды
     ChokeHold
         Длительность сна уменьшена с ~30 до ~15 секунд
     Slam
         Длительность оглушения с ~10 до ~3 секунд
     Restrain
         Длительность оглушения с ~9 до ~3 секунд
     Kick
        Длительность сна с ~27 до ~5 секунд
        Урон мозгу с 15 до 4
Sleeping Carp
-Цена уменьшена с 17 до 16
-Оглушение с ударов на харме зависящее от здоровья цели уменьшено с ~7 до ~3 секунд
-Отключается если владелец подвержен любой зависимости (начиная с никотина и заканчивая омнизином).
     Wrist Wrench
        Длительность оглушения с ~5 до ~0.5 секунд
     Head Kick
        Длительность оглушения с ~7 до ~3 секунд
     Back Kick
        Длительность оглушения с ~7 до ~3 секунд
Заклинания культа
     Shadow Shackle
        Теперь не действует на цель если она не в стамина крите, обычном крите или спит.
     Stun
        Длительность немоты увеличена до ~9 секунд
Генокрад
     Tentacle
        Теперь дает пассивный граб вместо мгновенного агрессивного
     Epinephrine Overdose
        Теперь synaptizine восстанавливает по 5 выносливости в тик

Commit:
Замена stun на weaken у глейра тени

Потому что паралич это очень РПшно, но ничерта не очевидно оглушена цель или нет.

Jordanstolethepreciousslam committed:
Merge branch 'testmerge' into АпдейтБоёвки

Zigamus commented on code/game/gamemodes/changeling/powers/mutations.dm:
```diff
@@ -260,7 +260,7 @@
 		var/obj/item/grab/G = C.grabbedby(src,1)
 		if(istype(G))
-			G.state = GRAB_AGGRESSIVE //Instant aggressive grab
+			G.state = GRAB_PASSIVE //Instant aggressive grab
```
//Instant aggressive grab (пассив)

Bizzonium replied:
```suggestion
			G.state = GRAB_PASSIVE
```

Commit:
Tweak: сила нелетального "огнестрела"

Стамина слаги
-Урон выносливости уменьшен с 66 до 55
Пули невидимого пистолета мима
-Урон выносливости уменьшен с 60 до 45
Пули трости мима
-Урон выносливости уменьшен с 60 до 45
Резиновая дробь
-Урон выносливости уменьшен с 20 до 15
Снаряды c-20r
-Урон выносливости уменьшен с 65 до 33
Экспансивные пули для стечкина
-Урон увеличен с 40 до 50

Commit:
Merge remote-tracking branch 'Jordan/АпдейтБоёвки' into АпдейтБоёвки

Bizzonium committed:
Update code/game/gamemodes/changeling/powers/mutations.dm
